include/internal/libfreenect2/protocol/NuiSensorLib.h

```diff
@@ -0,0 +1,1356 @@
+//--------------------------------------------------------------------------
```

Comment:
Why is this file needed in this repo? I looked around and it doesn't seem to be used anywhere. A reference to https://github.com/Microsoft/MixedRealityCompanionKit/blob/master/KinectIPD/NuiSensor/NuiSensorLib.h would suffice.

Reply:
I misunderstood the comment above to add a reference, so I've now removed the header.